specs/MultiProfile.md

```diff
@@ -245,6 +252,9 @@ interface ICoreWebView2Profile : IUnknown {
   /// Full path of the profile directory.
   [propget] HRESULT ProfilePath([out, retval] LPWSTR* value);
 
+  /// Get the cookie manager object for the profile.
+  [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
```

Comment:
ICoreWebView2Profile has already been released as experimental. So we won't be updating the existing ICoreWebView2Profile interface but rather making a new interface named ICoreWebView2Profile2. Although the experimental and staging versions don't derive from each other the public versions will and what we show in our API specs is the expected public interface.
```
interface ICoreWebView2Profile2 : ICoreWebView2Profile {
  /// Get the cookie manager object for the profile.
  [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
}
```

Reply:
Thanks for the explanation. Added _ICoreWebView2Profile2_ interface.